db/migrate/20170410164012_remove_workspaces_collection_foreign_key.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveWorkspacesCollectionForeignKey < ActiveRecord::Migration[5.0]
+  def change
+    remove_foreign_key :workspaces, :collection
```

Comment:
i think we need to change the model's relation to `has_many ... dependent: :nullify`, right?  See http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many